stats.js

```diff
@@ -0,0 +1,90 @@
+function pluck(property, collection) {
+  return collection.map(function(e){return e[property];});
+}
+function uniqueAuthors(collection) {
+  var seen = [];
+  var all = pluck('author', collection);
+  all.forEach(function(e) {
+    if (seen.indexOf(e) === -1) {seen.push(e);}
+  });
+  return seen;
+}
+function uniqueCategories(collection) {
+  var length = collection ?collection.length : 0;
+  var all = pluck('category', collection);
+  if (!length) {return [];}
+  var seen = [];
+  all.forEach(function(e) {
+    if (seen.indexOf(e) === -1) {seen.push(e);}
+  });
+  return seen;
+}
+function wordCount(collection) {
+  var all = pluck('markdown', collection);
+  var count = [];
+  all.forEach(function(e) {
+    var div = document.createElement("div");
```

Comment:
What was your reasoning for creating a div element here? Since the markdown source is not being rendered into HTML, you already have the markdown text available from your pluck function.